crates/uv-resolver/src/resolver/mod.rs

```diff
@@ -974,6 +1014,12 @@ impl<'a, Provider: ResolverProvider> Resolver<'a, Provider> {
                 }))
             }
 
+            Request::Installed(dist) => {
+                // TODO(zanieb): Forward the error from `metadata`
+                let metadata = dist.metadata().unwrap();
```

Comment:
This could block this pull request. We could add a new error variant here until `dist.metadata()` is refactored.

Reply:
What's the blocker? E.g., why does it need to be refactored?

Reply:
Unblocked in https://github.com/astral-sh/uv/pull/2596/commits/a8ba64f0d4a650ec7c9cc6bc1331f014103da3a4 which should clarify